Extensions/timestamps.css

```diff
@@ -39,13 +39,28 @@
 	margin-left: 20px;
 }
 
-.xtimestamp.xtimestamp_loading {
+.xkit--react .xtimestamp {
+	position: static;
+	color: var(--gray-40);
+	font-size: 14px;
+	line-height: 1em;
+	padding: 0 var(--post-padding);
+	margin-left: 0;
+	margin-top: calc(-1 * (var(--post-header-vertical-padding) / 2));
```

Comment:
?

does dividing by -2 not work on chrome?

Reply:
It does, I thought this would be a little more readable, but I can change it back to `/ -2` no problem.